### PR TITLE
postgresql flexible - deleting tables

### PR DESCRIPTION
**Pull request by SupDavid**
Hello @Lawouach , I had some problems with the previous pull request, I created a new one. 

I added the signature of my commits, I optimized the number of line and I added the line break at the level of the imports in the action file. 

I think we're good!

### CODE REVIEW

**Comment by SupDavid** [comment]
Hi @Lawouach  , I'm taking the liberty of getting back to you on this pull request, we need to integrate this feature into chaos toolkit for an operation next week and a few preparations are still needed.  Our timing is pretty tight...

I'd really appreciate it if you could make this happen quickly :)

File: chaosazure/postgresql_flexible/actions.py [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # -*- coding: utf-8 -*-
 import re
 
+import random
+import psycopg2
+
+from azure.identity import ClientSecretCredential
+from azure.mgmt.rdbms.postgresql_flexibleservers import PostgreSQLManagementClient
```
Comment (Lawouach) [inline chaosazure/postgresql_flexible/actions.py:8]: That should be:

```python
import re
import random

import pyscopg2
from azure.identity import ClientSecretCredential
```

But that's fine.

**Review by Lawouach — APPROVED** [review]

**Comment by Lawouach** [comment]
Hi @SupDavid 

I'll allow it for today but I'll have to fix a couple of things after merging. I think it's important you keep me updated ahead of time when you have deadlines :)

I'm on holiday so I'm taking time for this. I'd rather I knew a bit before hand so we could decide if there was a risk for you.

**Comment by Lawouach** [comment]
Hi @SupDavid and @jbblache 

I've merged and fixed the code linting (please consider running `pylama chaosazure` before pushing in the future). However the tests aren't passing.

Please send another PR fixing them so I can cut a release.

**Pull request merged**